collectors/python.d.plugin/am2320/README.md

```diff
@@ -0,0 +1,37 @@
+# AM2320
+This module will display a graph of the temperture and humity from a AM2320 sensor
+
+**Requirements:**
+ - Adafruit Circuit Python AM2320 library
+ - Adafruit AM2320 I2C sensor
+ - Python 3
+ - Raspberry Pi board
```

Comment:
Unless I'm mistaken, this should work with any system that can run Linux which supports the Adafruit Circuit Python module for the AM2320 sensor, correct?

If so, it would probably be better to remove this line from the requirements and update the configuration section to clarify that the instructions for for a Raspberry Pi.